JS/js-functions-2_arrow-functions/js/index.js

```diff
@@ -17,32 +31,53 @@ function getCurrentHour() {
   } else {
     return currentHour - 12 + "pm";
   }
-}
+};
 
-function getVectorLength(x, y, z) {
-  return (x ** 2 + y ** 2 + z ** 2) ** 0.5;
-}
+// function getVectorLength(x, y, z) {
+//   return (x ** 2 + y ** 2 + z ** 2) ** 0.5;
+// }
 
-function cleanInput(string) {
-  return string.toLowerCase().trim();
-}
+const getVectorLength = (x, y, z) => (x ** 2 + y ** 2 + z ** 2) ** 0.5;
 
+// function cleanInput(string) {
+//   return string.toLowerCase().trim();
+// }
+
+const cleanInput = (string) => string.toLowerCase().trim();
```

Comment:
```suggestion
const cleanInput = string => string.toLowerCase().trim();
```
you can remove () from string because the function takes only one parameter.